bgpd/bgp_vty.c

```diff
@@ -6745,6 +6745,11 @@ DEFPY (bgp_imexport_vrf,
 	safi_t safi;
 	afi_t afi;
 
+	if (import_name == NULL) {
+		vty_out(vty, "%% Missing import name\n");
+		return CMD_WARNING;
```

Comment:
Thanks for fixing this warning. Ideally we should use an assert here since `import_name` will never be NULL under normal conditions. In the long term we should try to make clippy generate these asserts for us, otherwise it's a lot of work to add asserts manually in each DEFPY for each non-optional parameter.